backend/src/repo/clientAppRepository.js

```
@@ -195,42 +194,68 @@ const getSessions = async (schema = DEFAULT_SCHEMA, userId) => {
+};
+
+const deleteExpiredSessions = async (schema = DEFAULT_SCHEMA) => {
+   const pool = await getPool(schema);
+   return sessionRepo.deleteExpiredSessions(pool);
+};
+
+export const userRepo = {
    createUser,
    getUsers,
    getUser,
    getUserByEmail,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'userRepo'. Consider to delete it or rename it.

'userRepo' is defined here:


(lint/suspicious/noRedeclare)